docs/api-reference/feeds.rst

```diff
@@ -21,6 +22,15 @@ newly created releases for individual projects on PyPI, including the project
 name and description, release version, and a link to the release page.
 
 
+Project Releases Feed
+---------------------
+
+Available at https://pypi.org/rss/project/<project>/releases.xml for each
```

Comment:
This link is not actually going to render, and since it's not going to resolve anyways, let's make it monospaced:
```suggestion
Available at ``https://pypi.org/rss/project/<project_name>/releases.xml`` for each
```